Sources/Datadog/RUM/AutoInstrumentation/Resources/URLSessionRUMResourcesHandler.swift

```diff
@@ -41,7 +41,7 @@ internal class URLSessionRUMResourcesHandler: URLSessionInterceptionHandler {
                 attributes: [:],
                 url: url,
                 httpMethod: RUMMethod(httpMethod: interception.request.httpMethod),
-                kind: RUMResourceType(request: interception.request),
+                kind: nil,
```

Comment:
i guess you forgot `nil` here.?

Reply:
And I'm wondering, why do we remove `RUMResourceType(request: interception.request)` here 🤔💭?

**The way it works:** 
We first try to deduce `kind` from request, and then read it from `response`. At the end, `request` has priority:
https://github.com/DataDog/dd-sdk-ios/blob/6fc6d5d8ed8b60794ca0e7bea7741dfc7bd991c3/Sources/Datadog/RUM/RUMMonitor/Scopes/RUMResourceScope.swift#L129

Reply:
I tried something and forgot to change this back